fastlane/lib/fastlane/actions/pod_lib_lint.rb

```diff
@@ -46,25 +51,29 @@ def self.details
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :use_bundle_exec,
-                                         description: "Use bundle exec when there is a Gemfile presented",
-                                         is_string: false,
-                                         default_value: true),
+                                       description: "Use bundle exec when there is a Gemfile presented",
+                                       is_string: false,
+                                       default_value: true),
           FastlaneCore::ConfigItem.new(key: :verbose,
-                                         description: "Allow output detail in console",
-                                         optional: true,
-                                         is_string: false),
+                                       description: "Allow output detail in console",
+                                       optional: true,
+                                       is_string: false),
           FastlaneCore::ConfigItem.new(key: :allow_warnings,
-                                         description: "Allow warnings during pod lint",
-                                         optional: true,
-                                         is_string: false),
+                                       description: "Allow warnings during pod lint",
+                                       optional: true,
+                                       is_string: false),
           FastlaneCore::ConfigItem.new(key: :sources,
-                                         description: "The sources of repos you want the pod spec to lint with, separated by commas",
-                                         optional: true,
-                                         is_string: false,
-                                         type: Array,
-                                         verify_block: proc do |value|
-                                           UI.user_error!("Sources must be an array.") unless value.kind_of?(Array)
-                                         end),
+                                       description: "The sources of repos you want the pod spec to lint with, separated by commas",
+                                       optional: true,
+                                       is_string: false,
+                                       type: Array,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Sources must be an array.") unless value.kind_of?(Array)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :swift_version,
+                                       description: "The SWIFT_VERSION that should be used to lint the spec. This takes precedence over a .swift-version file",
+                                       optional: true,
+                                       is_string: false),
```

Comment:
```suggestion
                                       is_string: true),
```

Can we change this to `true`? The test shows a string and I think making this true is safe and makes sense 👌